src/WebJobs.Extensions.OpenAI.Kusto/KustoSearchProvider.cs

```diff
@@ -169,29 +169,27 @@ KustoConnectionStringBuilder GetKustoConnectionString(string connectionName)
         return new KustoConnectionStringBuilder(connectionString);
     }
 
-    static string GetEmbeddingsString(ReadOnlyMemory<float> embedding)
+    static string GetEmbeddings(ReadOnlyMemory<float> embedding, bool returnString = false)
```

Comment:
nit: this parameter name is confusing because the method always returns a string. I think a better name would be `asJson` since I think that's really the distinction you're making. Also, I recommend *not* using default parameter values for private methods. It's mostly a feature designed for public-facing APIs.

Example:
```csharp
static string GetEmbeddingsString(ReadOnlyMemory<float> embedding, bool asJson)
```

Reply:
renamed to asJsonArray